index.bs

```diff
@@ -936,12 +935,12 @@ setInterval(() => {
 *This section is non-normative.*
 
 Receiving datagrams can be achieved by calling
-{{DatagramTransport/receiveDatagrams()}} method, remembering to check for null
-values indicating that packets are not being processed quickly enough.
+{{DatagramTransport/readableDatagrams}} attribute, remembering to check for
+null values indicating that packets are not being processed quickly enough.
```

Comment:
Note (no need to change anything in this PR): We never implemented this the null value feature in Blink. We should probably revisit #97 which discusses removing it.